kde_plasmoid_debian/Readme.md

```diff
@@ -0,0 +1,66 @@
+# Mycroft Plasmoid // Release 2.0
+#### Mycroft Ai Plasmoid and Skills for KDE Plasma 5 Desktop
+
+Auto Install Instructions:
+
+1. git clone https://github.com/MycroftAI/installers/kde_plasmoid_debian
```

Comment:
This is fine for now, but I want to experiment with a few alternatives, since this is a one-file installer.  I think we could either point to the file using the URL:
   https://raw.githubusercontent.com/MycroftAI/installers/master/kde_plasmoid_debian/install.sh

Or tell them to use wget instead of git (for really basic users who havent setup git).
Then change this to just:  `sudo bash install.sh` (skipping the chmod step).  33% fewer steps!

We can merge this first, then play with simplifying the instructions.